src/plugins/intel_cpu/src/nodes/executors/deconv.hpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu_memory.h"
+#include "onednn/iml_type_mapper.h"
+#include "executor.hpp"
+#include <common/primitive_hashing_utils.hpp>
+#include "nodes/common/dnnl_executor.h"
+
+namespace ov {
+namespace intel_cpu {
+
+struct DeconvKey {
+    DnnlMemoryDescCPtr inp0;
+    DnnlMemoryDescCPtr inp1;
+    DnnlMemoryDescCPtr bias;
+    DnnlMemoryDescCPtr out;
+
+    std::vector<ptrdiff_t> stride;
+    std::vector<ptrdiff_t> dilation;
+    ov::CoordinateDiff paddingL;
+    ov::CoordinateDiff paddingR;
+
+    bool isInt8;
+
+    dnnl::primitive_attr attr;
+    impl_desc_type implType;
+
+    size_t hash() const;
+    bool operator==(const DeconvKey& rhs) const;
+};
+
+// Defines way to add epsilon: inside sqrt or outside.
+struct DeconvAttrs {
```

Comment:
For some executors we have Attrs as past of the key, here we don't
Could you please clarify the reason?